middleware/kong/README.md

```diff
@@ -0,0 +1,57 @@
+# MIDDLEWARE KONG SignalFX detectors
+
+## How to use this module
+
+```hcl
+module "signalfx-detectors-middleware-kong" {
+  source      = "github.com/claranet/terraform-signalfx-detectors.git//middleware/kong?ref={revision}"
+
+  environment = var.environment
+  notifications = var.notifications
+}
+
+```
+
+## Purpose
+
+Creates SignalFX detectors with the following checks:
+
+- Kong heartbeat
+- Kong treatment limit
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:-----:|
+| detectors\_disabled | Disable all detectors in this module | `bool` | `false` | no |
+| environment | Infrastructure environment | `string` | n/a | yes |
+| filter\_custom\_excludes | Tags to exclude when using custom filtering (i.e "tag1:val1;tag2:val2") | `string` | `""` | no |
+| filter\_custom\_includes | Tags to filter signals on when custom filtering is used (i.e "tag1:val1;tag2:val2") | `string` | `""` | no |
+| filter\_use\_defaults | Use default filtering which follows tagging convention | `bool` | `true` | no |
+| heartbeat\_disabled | Disable all alerting rules for heartbeat detector | `bool` | n/a | yes |
+| heartbeat\_notifications | Notification recipients semicolon for every alerting rules of heartbeat detector | `string` | `""` | no |
+| heartbeat\_timeframe | Timeframe for system not reporting detector (i.e. "10m") | `string` | `"20m"` | no |
+| notifications | Notification recipients semicolon separated (i.e. "Email,my@mail.com;PagerDuty,credentialId") | `string` | n/a | yes |
+| prefixes\_slug | Prefixes list to prepend between brackets on every monitors names before environment | `list` | `[]` | no |
```

Comment:
Using _slug_ here feels wrong, that [word](https://en.wikipedia.org/wiki/Slug_(publishing)) is overused in this context IMO.